packages/model-viewer/src/features/animation.ts

```diff
@@ -98,6 +100,19 @@ export const AnimationMixin = <T extends Constructor<ModelViewerElementBase>>(
       }
     }
 
+    playOnce() {
```

Comment:
Considering the other features that were requested (number of loops, ping-pong) I think I'd prefer a more extensible API. Would you mind removing `playOnce` and instead adding a options object to the `play()` method? So we can  get `playOnce()` behavior  with `play({repetitions: 1})`, where `repetitions defaults to `Infinity`. Then we can easily  add other parameters to that  object without breaking anything.

Reply:
I thought of playOnce because I wanted to emphasize simplicity.
But I think your opinion is very true.